src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -118,25 +116,34 @@ namespace ts.codefix {
                         }
 
                     case SyntaxKind.NamespaceImport:
-                        const namespaceImport = <NamespaceImport>parent;
-                        if (namespaceImport.name === identifier && !(<ImportClause>namespaceImport.parent).name) {

```

Comment:
I believed ```namespaceImport.name !== identifier``` is an unreachable condition. So it seemed like an arbitrary extra check. If we have an identifier whose parent is a ```NamespaceImport```, then by definition ```namespaceImport.name``` has to be that identifier. Is that not right?